### DISCUSSION TITLE
Re-compare to keep the current position

### DISCUSSION THREAD

**Discussion by al20878** [WinMerge/winmerge#2413] [Ideas]
When the files get changed from the outside, can it be possible to have an option that would keep the current position after the files got re-compared?  Currently, WinMerge seems to always re-show the re-compared files from the very beginning. Thanks

**Comment by sdottaka** [comment]
What version of WinMerge are you using?

When WinMerge recompares changes made in an external application, it leaves the current location unchanged.

If you're using the latest version, please let us know the steps to reproduce.

**Comment by al20878** [comment]
Thank you for your response.  I am sure I am using the latest official release of WinMerge, and I did not do anything special -- just agreed for WinMerge to re-compare the files, when one was noted to have changed from the outside.  When the view was refreshed following that, the both files were shown from the top, which means that the position was lost.  Both files are on the local disk, and in fact, in the same directory.

BTW, it'd be very cool if the text on the Help / About WinMerge page could be selected with the mouse and copied to the clipboard.  It's currently totally "numb" to any mouse operations.  But here's the version:
2.16.42.1 X64

**Comment by al20878** [comment]
UPD: I tried to reproduce it myself, and I could not (the view stayed the same after re-comparison of the files).  But I def experienced the top view, which was why I submitted this request (files were quite large and I lost the current position, and that was quite frustrating)...  IDK if it was some sort of a glitch, though.

Still, it'd be great to have the version number copiable form the About page.  But that's a very minor issue.
